sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql

```diff
@@ -46,20 +46,17 @@ store_stats AS (
   ON
     country_name = name
 ),
-new_profiles_and_activations AS (
+_new_profiles AS (
   SELECT
     first_seen_date AS `date`,
-    country,
-    SUM(new_profile) AS new_profiles,
-    SUM(activated) AS activations,
+    first_reported_country AS country,
+    COUNT(*) AS new_profiles,
   FROM
-    firefox_ios.new_profile_activation
+    firefox_ios.firefox_ios_clients
```

Comment:
I think we wanted to use active user aggregates so that we wouldn't be impacted by shredder, righ?

Reply:
The ticket was specifying that this is the table we should be using and it should give us very close to AUA numbers 🤔 I did check `firefox_ios_clients` against `AUA` and the variance indeed was very small between the two. 30 or less difference between the two tables. Maybe we should have that discussion, right now we have three tables to calculate new_profiles and it's unclear to me why.

Reply:
This etl seems to be a full table rebuilt rebuild on every run and this will expose us to impacts of shredder. I think we can keep the source but make each run an incremental load of data partition by date. @kik-kik is this on the road map or the etl is going to stay full table rebuild going forward?

Reply:
yes, it's a small query so it's okay to just rebuild it every time. The potential problem I can see in converting this to an incremental query we run each day is that getting the data from apple is quite slow and would need to further investigate over time time to make sure we would feel confident enough that all the data for each day would be there by the time we run our incremental ETL. Currently, just looking at the results of the `store_stats` CTE it appears we potentially have a 4 day lag. Which might even be good make that more explicit, but the problem is we cannot even be certain that this is a fixed delay and that some days it might be 5 or 3 days delay before the data is available to us. We can look into this more, but I also think for now this is okay. 

Or at least if we use AUA as the source of the new_profile number then we should see pretty stable results.